### PR TITLE
Move duVal-Singularity recognition to src

### PR DESCRIPTION
**Pull request by afkafkafk13**
After this had been sitting in experimental for ages, being part of the regular CI-tests, I would like to move it to src now. 
The best place seems to be src/AlgebraicGeometry/Surfaces, as this is specifically for surfaces.

### CODE REVIEW

File: src/AlgebraicGeometry/AlgebraicGeometry.jl [hunk 1]
```diff
@@ -12,3 +12,4 @@ include("RationalPoint/Types.jl")
 include("RationalPoint/PointSet.jl")
 include("RationalPoint/AffineRationalPoint.jl")
 include("RationalPoint/ProjectiveRationalPoint.jl")
+include("Surfaces/duValSing.jl")
```
Comment (lgoettgens) [inline src/AlgebraicGeometry/AlgebraicGeometry.jl]:
```suggestion
include("Surfaces/duValSing.jl")

```

File: docs/src/AlgebraicGeometry/Surfaces/duValSing.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+
+```@meta
+CurrentModule = Oscar
+```
+
+# quick classifier/identifier for du Val singularities
```
Comment (simonbrandhorst) [inline docs/src/AlgebraicGeometry/Surfaces/duValSing.md]:
```suggestion
# Classifier/identifier for du Val singularities
```

Comment (afkafkafk13) [reply docs/src/AlgebraicGeometry/Surfaces/duValSing.md]: No, we also have a full classifier of singularities. This is a quick classifier on the basis of invariants.

Comment (afkafkafk13) [reply docs/src/AlgebraicGeometry/Surfaces/duValSing.md]: What about
`# Classifier/identifier specifically for du Val singularities`

Comment (fingolfin) [reply docs/src/AlgebraicGeometry/Surfaces/duValSing.md]: I am neutral on this, except that it should start with an uppercase letter :-)

Comment (simonbrandhorst) [reply docs/src/AlgebraicGeometry/Surfaces/duValSing.md]: Is this full classifier of singularities accessible and documented somewhere?

`# Classifier/identifier specifically for du Val singularities`
Looks good to me.

Comment (afkafkafk13) [reply docs/src/AlgebraicGeometry/Surfaces/duValSing.md]: > Is this full classifier of singularities accessible and documented somewhere?

Arnold's classifier is accessible in several implementations with slightly different capabilities in Singular as a library and as such accessible. I did not find any documentation on it though.
Yep, this should be put on the todo-list and after it is documented, a crossreference needs to be set here. 

> `# Classifier/identifier specifically for du Val singularities` Looks good to me.

Then I shall change it this way.

Comment (afkafkafk13) [reply docs/src/AlgebraicGeometry/Surfaces/duValSing.md]: done

**Review by simonbrandhorst — APPROVED** [review]
Thank you :-)

**Pull request merged**